lib/core.js

```diff
@@ -431,15 +439,6 @@ declare class Promise<R> {
     static reject<T>(error?: any): Promise<T>;
     static all<T, Elem: Promise<T> | T>(promises: Array<Elem>): Promise<Array<T>>;
     static race<T, Elem: Promise<T> | T>(promises: Array<Elem>): Promise<T>;
-
-    // Non-standard APIs common in some libraries
```

Comment:
Well, typescript 1.8 has now [augmenting to modules](https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#augmenting-globalmodule-scope-from-modules) that would, if implemented in Flow, solve this issue nicely, in my opinion :)

But I am not helping here much :)